src/main/java/bio/terra/service/snapshotbuilder/utils/ConceptChildrenQueryBuilder.java

```diff
@@ -45,17 +35,21 @@ public Query buildConceptChildrenQuery(
       SnapshotBuilderDomainOption domainOption, int parentConceptId) {
 
     // concept table and its fields concept_name and concept_id
-    TableVariable concept = TableVariable.forPrimary(TablePointer.fromTableName(CONCEPT));
-    FieldVariable conceptName = concept.makeFieldVariable(CONCEPT_NAME);
-    FieldVariable conceptId = concept.makeFieldVariable(CONCEPT_ID);
+    TableVariable concept =
+        TableVariable.forPrimary(TablePointer.fromTableName(ConceptConstants.CONCEPT));
+    FieldVariable conceptName = concept.makeFieldVariable(ConceptConstants.CONCEPT_NAME);
+    FieldVariable conceptId = concept.makeFieldVariable(ConceptConstants.CONCEPT_ID);
```

Comment:
This is OK for now, I don't think this PR is a great place to do any more than this level of refactoring. In fact, it might make sense to split out the string refactoring into its own PR so it's more apparent which changes are specific to the fix for DC-972.

But to give an idea about where I think this should go, I think we should have APIs more like:
```suggestion
    TableVariable concept = new Concept();
    FieldVariable conceptName = concept.name();
    FieldVariable conceptId = concept.id();
```

Reply:
Yes, I would love this, this is so much cleaner!!!

Reply:
Very much agreed - that seems like a great place to go with this.